presto-testng-services/src/main/java/io/prestosql/testng/services/FlakyTestRetryAnalyzer.java

```diff
@@ -96,16 +107,6 @@ public boolean retry(ITestResult result)
         return true;
     }
 
-    private static boolean isEnabled()
```

Comment:
I would prefer to keep this method. You might want to move logging here.

Reply:
i inlined the method becaause flow logging is generally handled by the caller.
i would end up logging twice when it's disabled